Client/Frontend/Settings/SearchBar/SearchBarSettingsViewModel.swift

```diff
@@ -104,8 +104,9 @@ private extension SearchBarSettingsViewModel {
         recordPreferenceChange(searchBarPosition)
 
         let notificationObject = [PrefsKeys.FeatureFlags.SearchBarPosition: searchBarPosition]
-        notificationCenter.post(name: .SearchBarPositionDidChange, object: notificationObject)
-    }
+        notificationProtocol.post(name: .SearchBarPositionDidChange, object: notificationObject)
+            }
+    
 
```

Comment:
Careful here with the indentation and extra spaces. You can make sure you have the right indentation by selecting the part of the code you're modifying and hitting the shortcut `ctrl+i`. We also have Swiftlint rules in our project to ensure there's only one space between functions. 
```suggestion
        notificationProtocol.post(name: .SearchBarPositionDidChange, object: notificationObject)
    }

```